go-controller/pkg/factory/factory.go

```diff
@@ -703,6 +750,10 @@ func getObjectMeta(objType reflect.Type, obj interface{}) (*metav1.ObjectMeta, e
 		if egressFirewall, ok := obj.(*egressfirewallapi.EgressFirewall); ok {
 			return &egressFirewall.ObjectMeta, nil
 		}
+	case DNSNameResolverType:
+		if dnsNameResolver, ok := obj.(*ocpnetworkapiv1alpha1.DNSNameResolver); ok {
+			return &dnsNameResolver.ObjectMeta, nil
+		}
```

Comment:
I think this and the following change are not required after we switched to using level-driven controllers?

Reply:
Removed the previous changes made to factory.go file for DNSNameResolver which are currently not required.